hooks/command

```diff
@@ -200,7 +203,10 @@ else
     else
       sleep "$job_status_retry_interval_sec"
       if [[ $timeout -gt 0 ]]; then
-        (( counter -= job_status_retry_interval_sec ))
+        if ! (( counter -= job_status_retry_interval_sec )); then
+          echo "Warning: never successfully got pod status"
+          exit 42
```

Comment:
```suggestion
          echo "Warning: timed out while checking init container status"
          break;
```
No need to exit here explicitly since we do that right after the loop exits